keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -671,22 +670,35 @@ type LimitedPriorityLevelConfiguration struct {
   // +optional
   AssuredConcurrencyShares int32
 
-  // `borrowablePercent` prescribes the fraction of the level's NominalCL that
+  // `lendablePercent` prescribes the fraction of the level's NominalCL that
   // can be borrowed by other priority levels.  This value of this
   // field must be between 0 and 100, inclusive, and it defaults to 0.
   // The number of seats that other levels can borrow from this level, known
-  // as this level's BorrowableConcurrencyLimit (BorrowableCL), is defined as follows.
+  // as this level's LendableConcurrencyLimit (LendableCL), is defined as follows.
   //
-  // BorrowableCL(i) = round( NominalCL(i) * borrowablePercent(i)/100.0 )
+  // LendableCL(i) = round( NominalCL(i) * lendablePercent(i)/100.0 )
   //
   // +optional
-  BorrowablePercent int32
+  LendablePercent int32
+  
+  // `borrowingLimitPercent`, if present, specifies a limit on how many seats
+  // this priority level can borrow from other priority levels.  The limit
+  // is known as this level's BorrowingConcurrencyLimit (BorrowingCL) and
+  // is a limit on the total number of seats that this level may borrow
+  // at any one time.  When this field is non-nil, it must hold a non-negative
+  // integer and the limit is calculated as follows.
+  //
+  // BorrowingCL(i) = round( NominalCL(i) * borrowingLimitPercent(i)/100.0 )
+  //
+  // When this field is left `nil`, the limit is effetively infinite.
+  // +optional
+  BorrowingLimitPercent *int32
```

Comment:
@MikeSpreitzer - I'm not opposed to introducing this concept, but I'm not really sure we should be exposing it now.

Do you have any usecase on your mind for it? As presented below, we will leave if unset for all default PLs, so I'm afraid that people will generally not be setting it.
And if that's the case, let's not introduce unnecessary complexity until we really need it...

@deads2k - for your thoughts

Reply:
You've summarized my reasoning better than I ever did myself.  Thanks Mike!

Reply:
Thanks Mike!
I suggest adding this explanation to the KEP itself - this is fairly useful explanation on its own.